supplier_api_tester/supplier_api_tester/tests/booking.py

```diff
@@ -141,3 +141,57 @@ def test_booking(api_url, api_key, product_id, timeslots: bool, version=1):
                 )
 
     return TestResult()
+
+@test_wrapper
+def test_cancellation(api_url, api_key, product_id, timeslots: bool, version=1):
+    '''Perform booking that will be cancelled'''
+    url = f'{api_url}/v{version}/products/{product_id}/reservation'
```

Comment:
As a first step check in the product catalog endpoint to see if the given product supports cancellation. If not we can skip this test and return a warning that we didn't check the cancellation.

Reply:
I think the response code already says that one cannot cancel such product

Reply:
That's true but I'm afraid that some suppliers that don't support cancellations at all wont even implement this endpoint.